include/tcframe/spec/core/StyleConfig.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <tuple>
+#include <utility>
+
+using std::move;
+using std::tie;
+
+namespace tcframe {
+
+struct StyleConfig {
+    friend class StyleConfigBuilder;
+
+private:
+    bool generateOutputFiles_;
+
+public:
+    bool generateOutputFiles() const {
```

Comment:
Hmm, I wouldn't use a verb like this for a boolean property.

How about `needsOutputFiles()`?